pkg/components/components.go

```diff
@@ -3,14 +3,14 @@ package components
 import (
 	"fmt"
 
-	cnav1alpha1 "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/v1alpha1"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	rbacv1 "k8s.io/api/rbac/v1"
 	extv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	opv1alpha1 "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/v1alpha1"
+	cnao "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/shared"
```

Comment:
maybe call it `shared` or `cnaoshared`? could be a bit confusing

Reply:
I think the point is to make everyone use the shared one instead of v1/v1alpha1, so I think `cnao` is a more appealing name.

Reply:
alrighty then